content/20.treatments.md

```diff
@@ -72,25 +72,35 @@ Differently from a DNA virus, which can use the host enzymes to propagate itself
+Furthermore, mice inoculated with Favipiravir was shown to increase survivalability. 
+In 2014, the drug was approved in Japan in order to treat patients infected with influenza and were resistant to conventional treatments, like neuraminidase inhibitors [@doi:10.2183/pjab.93.027].
 
 ##### Anticipated Mechanism
-
-Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase).
-While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
-Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1016/j.antiviral.2014.02.014].
-Furthermore, there are evidences that drug might also work against Corona virus infection.
-For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir [@doi:10.1016/j.eng.2020.03.007].
-The drug was tested on a sample of 80 patients (35 experimental sample, 45 control group) and increased the speed of recovery (measured as viral clearance from the patient by RT-PCR) of about 4 days, compared to the control sample treated with the other anti-virals, such as Lopinavir and Ritonavir.
-Also lung body scan seem to look better in about 91% of the patients analysed [@doi:10.1016/j.eng.2020.03.007].
-However, the size of the sample is too small to give a powerful statistics, as well as the choice of the patients did not take into consideration important factors such as previous clinical conditions, sex, while there was no age categorisation.
+Favipiravir (6-fluoro-3-hydroxy-2-pyrazinecarboxamide) acts as a purine and purine nucleoside analogue that inhibits viral RNA polymerase in a dose dependent manner across a range of RNA viruses, including Influenza virus [@doi:10.1128/AAC.49.3.981-986.2005; @doi:10.1128/AAC.01074-08; @doi:10.1128/AAC.00356-07; @doi:https://doi.org/10.1016/j.bbrc.2012.07.034; @doi:10.1128/AAC.01219-10]. 
+Evidence for T-705 inhibiting viral RNA polymerase are based on time-of-drug addition studies that found that viral loads were reduced with the addition of Favipiravir in early times post-infection [@doi:10.1128/AAC.49.3.981-986.2005; @doi:10.1016/j.bbrc.2012.07.034; @doi:10.1128/AAC.01219-10].
+Additionally, biochemical experiments showed that Favipiravir was recognized as a purine nucleoside analogue and incorporated into the viral RNA template. 
+A single incorporation does not influence RNA transcription, however multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
```

Comment:
```suggestion
A single incorporation of the analogue does not influence RNA transcription; however, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
```

Reply:
thank you for the suggestions, looks like you found the same ones as other reviewers!